### PR TITLE
refactor: Move project resource query, application publish item related api to dop

### PR DESCRIPTION
**Pull request by shuofan**
#### What this PR does / why we need it:
Move project resource query, application publish item related api to dop

#### Specified Reviewers:

/assign @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  core-services orchestrator decoupling         |
| 🇨🇳 中文    |   core-services orchestrator   解耦        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_update.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 var APP_PUBLISH_ITEM_RELATION_UPDATE = apis.ApiSpec{
 	Path:         "/api/applications/<application>/actions/update-publish-item-relations",
 	BackendPath:  "/api/applications/<application>/actions/update-publish-item-relations",
-	Host:         "core-services.marathon.l4lb.thisdcos.directory:9526",
+	Host:         "dop.marathon.l4lb.thisdcos.directory:9527",
```
Comment (Effet) [inline modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_update.go:25]: should move this file to dop subdir:

modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_update.go

File: modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_get.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 var APP_PUBLISH_ITEM_RELATION_GET = apis.ApiSpec{
 	Path:         "/api/applications/<application>/actions/get-publish-item-relations",
 	BackendPath:  "/api/applications/<application>/actions/get-publish-item-relations",
-	Host:         "core-services.marathon.l4lb.thisdcos.directory:9526",
+	Host:         "dop.marathon.l4lb.thisdcos.directory:9527",
```
Comment (Effet) [inline modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_get.go:25]: should move this file to dop subdir

File: modules/cmp/impl/addons/config.go [hunk 3]
```diff
@@ -146,7 +146,7 @@ func (a *Addons) ProjectQuotaCheck(identity apistructs.Identity, req apistructs.
 	preq.OrgID = uint64(oid)
 	preq.Name = p.Name
 
-	rsp, err := a.bdl.ListProject(identity.UserID, preq)
+	rsp, err := a.bdl.ListDopProject(identity.UserID, preq)
```
Comment (Effet) [inline modules/cmp/impl/addons/config.go:149]: will cmp depend to dop?

**Review by Effet — APPROVED** [review]

**Review by Effet — CHANGES_REQUESTED** [review]
.

File: modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_update.go (hunk 1 above)
Comment (shuofan) [reply modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_update.go:25]: done

File: modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_get.go (hunk 2 above)
Comment (shuofan) [reply modules/openapi/api/apis/core_services/cmdb_app_publish_item_relation_get.go:25]: done

File: modules/cmp/impl/addons/config.go (hunk 3 above)
Comment (shuofan) [reply modules/cmp/impl/addons/config.go]: fix legacy code, remove dependency

**Review by Effet — APPROVED** [review]

**Pull request merged**